### DISCUSSION TITLE
How long the Legacy Plan will lasts?

### DISCUSSION THREAD

**Discussion by rhlrjn2202** [swellstores/community#332] [Help] [answered]
I am on a legacy community plan, can someone tell me how this plan will last? Do I still get updates for my storefront?

**Comment by ericingram** [comment, marked as answer]
@rhlrjn2202 the community plan gets the same updates as all account, and although we may introduce new features for paid plans in the future, we do not intend to disable it for users that signed up before it was deprecated.
